templates/pools/pools.html

```diff
@@ -40,6 +66,9 @@
           <div id="performance-chart" class="w-100 mb-2 d-none">
             {{ template "performance_chart" .Data }}
           </div>
+          <div class="d-flex justify-content-center">
+            <a role="button" class="btn btn-primary d-none" id="performance-chart-btn-hide-all">hide all</a>
```

Comment:
Wrong font color, e.g. compare with the Sign Up button:
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/109136188/5638e1dc-9038-4120-b4cb-b2f8a74ea2d1)
![image](https://github.com/gobitfly/eth2-beaconchain-explorer/assets/109136188/76fbe53d-53f9-42bd-92b2-fe4f086cd2ad)